hack/gen-resolved-issue.py

```diff
@@ -620,6 +655,9 @@ def issue_details(record, issue_type, issue_description, issue_url):
         if None == os.environ.get('GOOGLE_APPLICATION_CREDENTIALS'):
```

Comment:
Not related to to this change, but other tools including Google's use `GOOGLE_SA_CREDENTIAL_FILE` for this (including Sippy).  Might be nice to be consistent